app/views/organisers/create.html.slim

```diff
@@ -0,0 +1 @@
+aa
```

Comment:
I need this because when the case fails and i do render index, it gives me error empty:nil class when is display list of organiser again. Ill look into the error now.